src/components/screens/IndexScreen/IndexScreen.js

```diff
@@ -240,14 +319,82 @@ export function PureIndexScreen({ ...props }) {
         logo={GitlabLogoSVG}
       />
 
-      <CTA
-        text={<span>Storybook is quick to install and it’s easy to get started.</span>}
-        action={
-          <Button appearance="secondary" isLink href={docs}>
-            Go to the docs
-          </Button>
-        }
-      />
+      <Contrast>
+        <Separator />
+        <ValueProp
+          orientation="right"
+          media={
+            <Placeholder ratio={0.75}>
+              <Video
+                src="videos/storybook-publish-review-greybg-optimized-lg.mp4"
+                alt="Publish to review workflow video"
+                shouldChangeSize
+              />
+            </Placeholder>
+          }
+          title="Share to get sign off from teammates"
+          desc="Stories show how UIs actually work not just how they're supposed to work. That makes
+            gathering feedback and reproductions easy."
+          lazyloadPlaceholder={<Placeholder ratio={0.75} />}
+        />
+        <BenefitList>
+          <BenefitItem
+            image={<img src="/images/home/share-publish.png" alt="Publish Storybook online" />}
+            title="Publish to get feedback"
+            desc={
+              <>
+                <Link href={`${docs}react/sharing/publish-storybook`}>Publish Storybook</Link>{' '}
+                online to give your team a universal reference point for feedback.
+              </>
+            }
+          />
+          <BenefitItem
+            image={<img src="/images/home/share-embed.png" alt="Embed stories" />}
+            title="Embed stories in your favorite sites"
+            desc={
+              <>
+                <Link href={`${docs}react/sharing/embed`}>Embed stories</Link> to showcase your work
```

Comment:
This feels quite similar to the first point (publish to get feedback). How about:

> Embed stories to showcase live interactive examples in your docs. Works with Notion, Medium and any site supporting the oEmbed standard.

Reply:
👍 sounds good